src/utils/markdown.ts

```diff
@@ -39,41 +39,75 @@ export function generateMarkdown(results: TestResult[]) {
   );
 }
 
+export function generateSimplifiedMarkdown(results: TestResults, outputFile: string) {
+  const markdownFile = new MarkdownFile();
+
+  markdownFile.startTable();
+  markdownFile.addTestResultsToTable(results);
+  markdownFile.endTable();
+
+  writeFileSync(
+    outputFile,
+    markdownFile.toString(),
+    "utf-8"
+  );
+}
+
 class MarkdownFile {
   private content: string[] = [];
 
-  constructor(title: string, sidebarTitle: string) {
-    this.content.push(
-      "---",
-      `title: ${title}`,
-      `sidebar_title: ${sidebarTitle}`,
-      "---",
-      "",
-      "The following open-source GraphQL server libraries provide support for Apollo Federation and are included in our test suite.",
-      ""
-    );
+  constructor(title?: string, sidebarTitle?: string) {
+    if (title && sidebarTitle) {
+      this.content.push(
+        "---",
+        `title: ${title}`,
+        `sidebar_title: ${sidebarTitle}`,
+        "---",
+        "",
+        "The following open-source GraphQL server libraries provide support for Apollo Federation and are included in our test suite.",
+        ""
+      );
+    }
   }
 
-  startLanguageTable(result: TestResult) {
-    this.content.push("", `## ${result.language}`, "");
+  startTable(result?: TestResultDetails) {
+    let headers = "<th>Federation 1 Support</th><th>Federation 2 Support</th>"
+    if (result) {
+      this.content.push("", `## ${result.language}`, "");
+      headers = "<th width=\"300\">Library</th>" + headers;
```

Comment:
nbd: backticks/single quotes are nice for strings that use double quotes
```suggestion
      headers = '<th width="300">Library</th>' + headers;
```